debian/rdma-core.install

```diff
@@ -22,6 +22,7 @@ lib/udev/rules.d/90-rdma-ulp-modules.rules
 lib/udev/rules.d/90-rdma-umad.rules
 usr/lib/truescale-serdes.cmds
 usr/sbin/iwpmd
+usr/sbin/rdma_topo
```

Comment:
Can we name it rdma-topo and not rdma_topo?

Also why did you put it in kernel-boot directory? I would expect that you will create new directory tools/ and put it there.

Reply:
Most of the tools we have in bin use _ not -
I put it there because it is mainly about writing out the kernel command line and didn't think another directory was really needed.